File: SDKs/Aspose.Slides_Cloud_SDK_for_CSharp/src/Com/Aspose/Slides/Model/DocumentProperty.cs
using System;
using System.Text;
using System.Collections;
using System.Collections.Generic;

namespace Com.Aspose.Slides.Model {
  public class DocumentProperty {
    public string Name { get; set; }

    public string Value { get; set; }

    public bool? BuiltIn { get; set; }

    public ResourceUri SelfUri { get; set; }

    public List<ResourceUri> AlternateLinks { get; set; }

    public List<ResourceUri> Links { get; set; }

    public override string ToString()  {
      var sb = new StringBuilder();
      sb.Append("class DocumentProperty {\n");
      sb.Append("  Name: ").Append(Name).Append("\n");
      sb.Append("  Value: ").Append(Value).Append("\n");
      sb.Append("  BuiltIn: ").Append(BuiltIn).Append("\n");
      sb.Append("  SelfUri: ").Append(SelfUri).Append("\n");
      sb.Append("  AlternateLinks: ").Append(AlternateLinks).Append("\n");
      sb.Append("  Links: ").Append(Links).Append("\n");
      sb.Append("}\n");
      return sb.ToString();
    }
  }
  }

File: SDKs/Aspose.Slides-Cloud-SDK-for-Java/src/main/java/com/aspose/slides/model/DocumentProperty.java
package com.aspose.slides.model;

public class DocumentProperty {
  private String Name = null;
  private String Value = null;
  private Boolean BuiltIn = null;
  /**
	 * getName
	 * Gets String
	 * @return Name
	 */
  public String getName() {
    return Name;
  }

	/**
	 * setName
	 * Sets String
	 * @param Name String
	 */
  public void setName(String Name) {
    this.Name = Name;
  }

  /**
	 * getValue
	 * Gets String
	 * @return Value
	 */
  public String getValue() {
    return Value;
  }

	/**
	 * setValue
	 * Sets String
	 * @param Value String
	 */
  public void setValue(String Value) {
    this.Value = Value;
  }

  /**
	 * getBuiltIn
	 * Gets Boolean
	 * @return BuiltIn
	 */
  public Boolean getBuiltIn() {
    return BuiltIn;
  }

	/**
	 * setBuiltIn
	 * Sets Boolean
	 * @param BuiltIn Boolean
	 */
  public void setBuiltIn(Boolean BuiltIn) {
    this.BuiltIn = BuiltIn;
  }

  @Override
  public String toString()  {
    StringBuilder sb = new StringBuilder();
    sb.append("class DocumentProperty {\n");
    sb.append("  Name: ").append(Name).append("\n");
    sb.append("  Value: ").append(Value).append("\n");
    sb.append("  BuiltIn: ").append(BuiltIn).append("\n");
    sb.append("}\n");
    return sb.toString();
  }
}

